app/frontend/testnet/libs/bech32.js

```diff
@@ -0,0 +1,32 @@
+import bech32 from "bech32";
+
+// Note(ppershing): yeah, really, bech32 lib has this stupid api
+export const bech32_encode = (prefix, data) => {
+  const words = bech32.toWords(data);
+  return bech32.encode(prefix, words, 1000);
```

Comment:
add a note that we need longer than default length for privkeys and 1000 should suffice